integration_tests/src/test/resources/cases/dump_shards_android.yml

```diff
@@ -0,0 +1,3 @@
+gcloud:
```

Comment:
I think we should add configuration with actual sharding applied, 2-3 shards should be enough.